src/snadmin/install-demo/import/System/Schema/ContentTypes/GroupCtd.xml

```diff
@@ -3,11 +3,71 @@
   <Description>$Ctd-Group,Description</Description>
   <Icon>Group</Icon>
   <Fields>
+    <Field name="Name" type="ShortText">
+      <DisplayName>$Ctd-Group,Name-DisplayName</DisplayName>
+      <Description>$Ctd-Group,Name-Description</Description>
+      <Configuration>
+        <DefaultOrder>80</DefaultOrder>
+        <VisibleBrowse>Hide</VisibleBrowse>
+        <VisibleEdit>Show</VisibleEdit>
+        <VisibleNew>Hide</VisibleNew>
+        <Compulsory>true</Compulsory>
+        <MaxLength>20</MaxLength>
+        <ControlHint>sn:ShortText</ControlHint>
+      </Configuration>
+    </Field>
+    <Field name="LastSync" type="DateTime">
+      <DisplayName>$Ctd-Group,LastSync-DisplayName</DisplayName>
+      <Description>$Ctd-Group,LastSync-Description</Description>
+      <Configuration>
+        <DefaultOrder>70</DefaultOrder>
+        <DateTimeMode>DateAndTime</DateTimeMode>
+        <VisibleBrowse>Hide</VisibleBrowse>
+        <VisibleEdit>Show</VisibleEdit>
```

Comment:
Is it necessary here to show lastsync field on the edit view? It is by default read-only, so i think it should be set to be hidden